e2e_tests/conftest.py

```diff
@@ -50,3 +70,42 @@ def basic_bentoservice_v2():
     bento_name = f'{bento_svc.name}:{bento_svc.version}'
     yield bento_name
     delete_bento(bento_name)
+
+
+@pytest.fixture()
+def temporary_docker_postgres_url():
+    ensure_docker_available_or_raise()
+    temp_dir = TempDirectory()
+    temp_dir.create()
+    container_name = 'e2e-yatai-pg-docker'
+
+    command = [
+        'docker',
+        'run',
+        '--rm',
+        '--name',
+        container_name,
+        '-e',
+        'POSTGRES_PASSWORD=postgres',
+        '-p',
+        '5432:5432',
```

Comment:
this will break if we have multiple tests using this fixture and running concurrently right?
maybe make this fixture `scope=module`?